### ISSUE TITLE
Environment forwards the wrong version

### ISSUE THREAD

**Issue by passsy** [pingbird/puro#19]
Setup: Create a new environment using a different flutter version than the global one

```bash
$ puro create three_seven

$ puro upgrade three_seven 3.7.12
[✓] Upgraded environment `three_seven`
    stable / 3.10.6 / f468f3366c => stable / 3.7.12 / 4d9e56e694
```

```bash
$ puro ls
[i] Environments:
    * stable      (stable / 3.10.5 / 796c8ef792)
      beta        (not installed)
      master      (not installed)
      three_seven (stable / 3.7.12 / 4d9e56e694)
```

Then the `flutter` binary of the `three_seven` environment reports the wrong version

```bash
$ cd .puro/envs/three_seven/flutter/
$ ./bin/flutter doctor --version
Flutter 3.10.5 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 796c8ef792 (5 weeks ago) • 2023-06-13 15:51:02 -0700
Engine • revision 45f6e00911
Tools • Dart 3.0.5 • DevTools 2.23.1
```

**Comment by pingbird** [comment]
This should be fixed in 1.3.2, thanks for the bug report!

**Comment by passsy** [comment]
Awesome, version detection in IntelliJ works now flawlessly :)
